### PR TITLE
Drop unused stream/sstream includes in main/utils

### PR DESCRIPTION
**Pull request by dnnr**

### CODE REVIEW

File: ci-build.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def tox(tox_args, build_dir):
     sp.check_call('fix_include --dry_run --sort_only --reorder /hlwm/*/*.{h,cpp,c}', shell=True, executable='bash')
 
     # Run include-what-you-use (just printing the result for now)
-    sp.check_call(f'iwyu_tool -p . -j "$(nproc)" -- --mapping_file=/hlwm/.hlwm.imp', shell=True, cwd=build_dir)
+    sp.check_call(f'iwyu_tool -p . -j "$(nproc)" -- --verbose=3 --transitive_includes_only --mapping_file=/hlwm/.hlwm.imp', shell=True, cwd=build_dir)
```
Comment (t-wissmann) [inline ci-build.py]: Was this change on purpose (in this PR)?

Comment (dnnr) [reply ci-build.py]: Oh, not at all. Thanks for noticing. Removed it.
